packages/site/static.config.js

```diff
@@ -4,12 +4,12 @@ import constants from './src/constants';
 import Document from './src/html';
 
 const isStaging = process.env.REACT_STATIC_STAGING === 'true';
-const basePath = 'open-source/urql';
 
 export default {
   plugins: [
     resolve(__dirname, 'plugins/monorepo-fix/'),
-    resolve(__dirname, 'plugins/preact'),
+    resolve(__dirname, 'plugins/react-router/'),
+    // resolve(__dirname, 'plugins/preact/'),
```

Comment:
uncomment?

Reply:
oh, I haven't tested this yet; give me a second